### PR TITLE
Add symfony 5 support

### PR DESCRIPTION
**Pull request by acrobat**

### CODE REVIEW

**Comment by chq81** [comment]
@acrobat Hi there. Any idea when the acl-bundle gets the symfony 5 upgrade?

**Comment by fabpot** [comment]
Tests must be fixed first.

**Comment by acrobat** [comment]
Yes I started the pullrequest a while ago, but I'm planning on finishing it in the coming days!

**Comment by acrobat** [comment]
Yes I started the pullrequest a while ago, but I'm planning on finishing it

@fabpot would you be ok with dropping support for php 5.6?

**Comment by fabpot** [comment]
Sure, you can probably make a new major version that only supports Symfony 4.4/5.0 as well.

**Comment by acrobat** [comment]
@fabpot Ok, will do!

**Comment by acrobat** [comment]
@fabpot I've dropped the old php and symfony versions and sf5 support is added. I've also fixed the command return code deprecation.

/cc @chq81

**Comment by chq81** [comment]
@acrobat Thanks a lot.

File: composer.json [hunk 1]
```diff
@@ -39,7 +39,6 @@
     "autoload-dev": {
         "psr-4": { "Symfony\\Bundle\\AclBundle\\Tests\\": "tests/" }
     },
-    "minimum-stability": "dev",
     "extra": {
         "branch-alias": {
             "dev-master": "1.0-dev"
```
Comment (fabpot) [inline composer.json]: Can you bump the version to 2.0-dev?

Comment (acrobat) [reply composer.json]: Fixed!

**Comment by fabpot** [comment]
Thank you @acrobat.

**Comment by chq81** [comment]
@fabpot Thanks a lot. Would you mind tagging a new release?

**Comment by fabpot** [comment]
Done

**Comment by chq81** [comment]
@fabpot Thank you!

**Pull request merged**